vital/config/config_block_io.h

```diff
@@ -258,6 +258,9 @@ application_config_file_paths(std::string const& application_name,
  *
  * \return
  *   List of additional application configuration search paths.
+ *
+ * \note This is the only function that will not guess the install prefix if
```

Comment:
This comment is not clear and maybe wrong.  I suppose you are referring to not guessing the `app_install_prefix` if left empty?  It does still guess the `kwiver_install_prefix` if left empty.

It seems we might want to update the documentation for the other functions to say that if `install_prefix` is left blank then a prefix is guessed based on the location of the calling executable.

Reply:
>  It does still guess the `kwiver_install_prefix` if left empty.

AFAICT that's not true, but maybe I'm missing something. Where are you seeing that?

Reply:
Okay it seems I may have misread the code.

The more important point is that it doesn't make sense to document that this function will not guess the prefix without documenting the other function that do guess the prefix and stating that the guess is based on the executable location.